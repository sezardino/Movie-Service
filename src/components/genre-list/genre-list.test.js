import React from 'react';
import renderer from 'react-test-renderer';
import GenreList from './genre-list.jsx';

it(`GenreList are rendered`, () => {
	const arr = [
		{key: `all`, title: `All genres`},
		{key: `comedy`, title: `Comedies`},
		{key: `crime`, title: `Crime`},
	];
	const tree = renderer.create(<GenreList genres={arr} />).toJSON();

	expect(tree).toMatchSnapshot();
});
