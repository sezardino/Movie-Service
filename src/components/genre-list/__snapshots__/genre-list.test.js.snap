// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GenreList are rendered 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      All genres
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Comedies
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Crime
    </a>
  </li>
</ul>
`;
